hal/src/hal-sdcard.adb

```diff
@@ -0,0 +1,382 @@
+with Ada.Text_IO; use Ada.Text_IO;
```

Comment:
Well, as already told live, but for the record:

I2C or SPI protocols are 100% handled by the hardware. Here we're in a situation a bit similar to Bluetooth, where the software is supposed to initialise the sd/mmc card according to a certain standardised protocol, just so that the handshake between the controller on the card and the host can discuss properly (and efficiently). I2C or SPI don't need that.